elastic/kibana
# [Uptime] index default severity of warning for tls and monitor status alerts

## Summary

Fixes #108443 

Adds severity of warning as the default severity for Uptime monitor status and TLS alerts.

<img width="1524" alt="Screen Shot 2021-08-16 at 11 56 59 AM" src="https://user-images.githubusercontent.com/11356435/129598890-889b2d92-9566-48ec-b621-991799d94fad.png">

### Testing

1. Start heartbeat, ensuring that you have a few monitors that are always down and a few monitors that are configured with expiring certificates.
2. Navigate to Uptime and create a monitor status alert and a TLS alert by using the alerts dropdown in the heading.
3. Navigate to the Observability alerts table and ensure that the severity displays as Warning.
